src/main/java/bookstore/dto/user/UserResponseDto.java

```diff
@@ -0,0 +1,14 @@
+package bookstore.dto.user;
+
+public record UserResponseDto(
+        Long id,
```

Comment:
```suggestion
        Long id,
        
```
I'm not a fan of empty lines, but if you like it, stick to the same style in all your classes